recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ outputs:
       run:
         - python >={{ python_min }},<3.13
         - openff-toolkit-base >=0.11.0
-        - openff-interchange-base ~=0.3.18
+        - openff-interchange-base =0.4
```

Comment:
I want this because "out with the old, in with the new" makes my life easier, but I don't want to do that at the cost of preventing ForceBalance-based fits